lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -505,7 +505,7 @@ sub overview {
 
 sub latest {
     my ($self) = @_;
-    my %search_args;
+    my %search_args = (limit => 1);
```

Comment:
Without limit => 1

```
SELECT me.id, me.slug, me.result_dir, me.state, me.priority, me.result, me.clone_id, me.retry_avbl, me.backend, me.backend_info, me.TEST, me.DISTRI, me.VERSION, me.FLAVOR, me.ARCH, me.BUILD, me.MACHINE, me.group_id, me.t_started, me.t_finished, me.t_created, me.t_updated, settings.id, settings.key, settings.value, settings.job_id, settings.t_created, settings.t_updated, parents.child_job_id, parents.parent_job_id, parents.dependency, children.child_job_id, children.parent_job_id, children.dependency FROM jobs me LEFT JOIN job_settings settings ON settings.job_id = me.id LEFT JOIN job_dependencies parents ON parents.child_job_id = me.id LEFT JOIN job_dependencies children ON children.parent_job_id = me.id WHERE ( me.TEST = ? ) ORDER BY me.id DESC: 'foobar'
    ok 10 - GET http://localhost:56979/tests/latest?test=foobar
```

With it:

```
SELECT me.id, me.slug, me.result_dir, me.state, me.priority, me.result, me.clone_id, me.retry_avbl, me.backend, me.backend_info, me.TEST, me.DISTRI, me.VERSION, me.FLAVOR, me.ARCH, me.BUILD, me.MACHINE, me.group_id, me.t_started, me.t_finished, me.t_created, me.t_updated, settings.id, settings.key, settings.value, settings.job_id, settings.t_created, settings.t_updated, parents.child_job_id, parents.parent_job_id, parents.dependency, children.child_job_id, children.parent_job_id, children.dependency FROM (SELECT me.id, me.slug, me.result_dir, me.state, me.priority, me.result, me.clone_id, me.retry_avbl, me.backend, me.backend_info, me.TEST, me.DISTRI, me.VERSION, me.FLAVOR, me.ARCH, me.BUILD, me.MACHINE, me.group_id, me.t_started, me.t_finished, me.t_created, me.t_updated FROM jobs me WHERE ( me.TEST = ? ) ORDER BY me.id DESC LIMIT ?) me LEFT JOIN job_settings settings ON settings.job_id = me.id LEFT JOIN job_dependencies parents ON parents.child_job_id = me.id LEFT JOIN job_dependencies children ON children.parent_job_id = me.id WHERE ( me.TEST = ? ) ORDER BY me.id DESC: 'foobar', '1', 'foobar'
    ok 10 - GET http://localhost:38912/tests/latest?test=foobar
```
I assume the prefetch breaks the first, so it actually create a cursor and then stops after retrieving the first row